### ISSUE TITLE
"wicket:id reference problem" when using Wicket 7 queue()

### ISSUE THREAD

**Issue by thorbolo** [minman/wicketforge#170]
queue(Component) allows you to add Components independent from the markup hierarchy.

Would be nice if Wicketforge gets that and matches it.

Maybe this relates to #99

**Comment by minman** [comment]
Yes, currently wicket 7 queuing is not supported. I will add support for it but it's lot to do, so it can take a while.

**Comment by dutrieux** [comment]
Hello @bamarc
It will nice to have this functionnality, no complete at first time, but at least, the switch class/markup when we click on markupid, and if there is markup id ambiguities, propose a list to choose one.

Best regards
